services/web/server/src/simcore_service_webserver/conversations/_conversation_message_service.py

```diff
@@ -139,12 +137,12 @@ async def _create_support_message_with_first_check(
         Tuple containing the created message and whether it's the first message
     """
 
-    @exclusive(
-        get_redis_lock_manager_client_sdk(app),
-        lock_key=CONVERSATION_MESSAGE_REDIS_LOCK_KEY.format(conversation_id),
-        blocking=True,
-        blocking_timeout=None,  # NOTE: this is a blocking call, a timeout has undefined effects
-    )
+    # @exclusive(
```

Comment:
If you want to remove it. just do so. please do not leave commented code.  If you are not sure, you can always enable/disable dynamically with a varaible